packages/terra-table/src/Table.jsx

```diff
@@ -291,19 +299,28 @@ function Table(props) {
         role={gridContext.role}
         aria-labelledby={ariaLabelledBy}
         aria-label={ariaLabel}
-        className={cx('table', theme.className)}
+        className={cx('table', theme.className, { headerless: !hasColumnHeaders })}
         {...(activeIndex != null && { onMouseUp, onMouseMove, onMouseLeave: onMouseUp })}
       >
         <ColumnContext.Provider
           value={columnContextValue}
         >
+          <colgroup>
+            {tableColumns.map((column) => (
+              // eslint-disable-next-line react/forbid-dom-props
+              <col style={{ width: `${column.width}px` }} />
```

Comment:
I guess we need to add a `key` prop:
<img width="764" alt="Screenshot 2023-10-19 at 10 57 51 AM" src="https://github.com/cerner/terra-framework/assets/119358186/8c25c0ba-5918-488a-912a-b7597803df0b">